IdentityCore/src/webview/systemWebview/ios/MSIDSafariViewController.m

```diff
@@ -67,7 +67,13 @@ - (instancetype)initWithURL:(NSURL *)url
         _startURL = url;
         _context = context;
         
+#if TARGET_OS_UIKITFORMAC
+        __auto_type config = [SFSafariViewControllerConfiguration new];
+        config.entersReaderIfAvailable = NO;
```

Comment:
The default value is NO, so not sure if we need to even create config here.
Also, can we use same API on iOS? Seems to be available since iOS 11. We still support iOS 10, but we'll be dropping it some time after iOS 13 is released.

Reply:
Updated with @available(iOS 11.0, *). We can remove this after we drop iOS 10.